tests/components/evohome/test_init.py

```diff
@@ -20,6 +24,9 @@ async def test_entities(
 ) -> None:
     """Test entities and state after setup of a Honeywell TCC-compatible system."""
 
+    # some of the extended state attrs are relative the current time
+    async_fire_time_changed(hass, datetime(2024, 7, 10, 12, 0, 0))
```

Comment:
Do we need to freeze time or is this enough?

Reply:
This is not enough - the new dt is not sticking. 

I am still trying to work out what fixture to use.

Reply:
Try using the `freezer` fixture.